CorsixTH/CorsixTH.lua

```diff
@@ -56,13 +56,16 @@ function dofile(name)
   return save_results(done_files, name, persist.dofile(code_dir .. name .. ".lua"))
 end
 
+-- Adds code dir as a first place to look for required files
+package.path = code_dir .. "?.lua" .. ";" .. package.path
```

Comment:
also add code_dir .. "?" .. pathsep .. "init.lua" for completeness - lua normally allows that naming convention.